test/synapsebridge/buildBridgeTokenTransaction-test.ts

```diff
@@ -178,6 +178,16 @@ describe("SynapseBridge - buildBridgeTokenTransaction tests", function(this: Moc
         makeTestCase(Tokens.DFKTEARS,   Tokens.DFKTEARS,  ChainId.DFK,       ChainId.HARMONY,     redeem),
         makeTestCase(Tokens.H20,        Tokens.H20,       ChainId.ETH,       ChainId.POLYGON,     deposit),
         makeTestCase(Tokens.H20,        Tokens.H20,       ChainId.POLYGON,   ChainId.ETH,         redeem),
+        makeTestCase(Tokens.SFI,        Tokens.SFI,       ChainId.ETH,       ChainId.AVALANCHE,   deposit),
+        makeTestCase(Tokens.SFI,        Tokens.SFI,       ChainId.AVALANCHE, ChainId.ETH,         redeem),
+        makeTestCase(Tokens.WBTC,       Tokens.WBTC,      ChainId.ETH,       ChainId.KLAYTN,      deposit),
+        makeTestCase(Tokens.WBTC,       Tokens.WBTC,      ChainId.KLAYTN,    ChainId.ETH,         redeem),
+        makeTestCase(Tokens.USDC,       Tokens.USDC,      ChainId.ETH,       ChainId.KLAYTN,      deposit),
```

Comment:
Let's add additional test cases for the stablecoins to / from Klaytn since it's the first time we're bridging stablecoins in a non-nUSD way